A55-xds-stateful-session-affinity.md

```diff
@@ -362,28 +382,35 @@ for the `xds_override_host_experimental` policy will be defined as follows:
 ```proto
 // Configuration for the override_host LB policy.
 message OverrideHostLoadBalancingPolicyConfig {
-  // if set to false this LB policy will be a no-op
-  bool enabled = 1;
+  enum HealthStatus {
+    UNKNOWN = 0;
+    HEALTHY = 1;
+    UNHEALTHY = 2;
```

Comment:
We don't need UNHEALTHY, TIMEOUT, or DEGRADED here, since we're not going to support them.

Reply:
fixed